m4/ax_cxx_compile_stdcxx_11.m4

```diff
@@ -0,0 +1,142 @@
+# ============================================================================
```

Comment:
Isn't the newer macro to use `AX_CXX_COMPILE_STDCXX` with a wrapper for the specific standard like ` AX_CXX_COMPILE_STDCXX_11`? That's what I've been using the last several years in own projects, I thought this standalone one was deprecated/obsolete or something.